### PR TITLE
Added TextSymbolizer handling in SLD util and styler

### PR DESCRIPTION
**Pull request by weskamm**
This adds support for styling TextSymbolizers in the SLD Styler.

### CODE REVIEW

File: src/view/container/SLDStyler.js [hunk 1]
```diff
@@ -141,6 +160,14 @@ Ext.define('BasiGX.view.container.SLDStyler', {
              * would list all fonts from the GEOSERVER_DATA_DIR/fonts directory
              */
             geoserverFontListUrl: null,
+
+            /**
+             * The REST URL of the GeoServer to retrieve all installed fonts.
+             * E.g. http://localhost:8080/geoserver/rest/fonts
+             * would list all fonts available in GoeServer
```
Comment (annarieger) [inline src/view/container/SLDStyler.js]: absolutely minor: typo `GeoServer`

File: src/view/container/SLDStyler.js [hunk 2]
```diff
@@ -882,6 +927,327 @@ Ext.define('BasiGX.view.container.SLDStyler', {
...
+        if (this.getMode() === 'line') {
+            fs.items.push({
+                xtype: 'numberfield',
+                bind: {
+                    fieldLabel: '{textPerpendicularOffsetLabel}'
+                },
+                value: perpendicularOffset,
+                name: 'perpendicularoffset',
+                minValue: -500,
+                maxValue: 500,
```
Comment (annarieger) [inline src/view/container/SLDStyler.js]: Shouldn't here further configs like `allowDecimals`, `decimalPrecision` or `step` be added as well as configured below for other numberfields?

Comment (weskamm) [reply src/view/container/SLDStyler.js]: No, as the parameters behave differently (different units).

**Comment by weskamm** [comment]
Thanks for review, merging

**Pull request merged**